docs/parameters.md

```diff
@@ -13,18 +56,61 @@ When selecting the bounds for a parameter it is important to remember that all v
     max: 4000
 ```
 
-You would consume this parameter in a `spec.containers[].resources` patch of:
+This parameter would be consumed in a patch of a Deployment's `spec.template.spec.containers[].resources` field:
 
 ```yaml
-  requests:
-    cpu: "{{ .Values.cpu }}m"
+  patches:
+  - patch: |
+      spec:
+        template:
+          spec:
+            containers:
+            - name: container-name
+              resources:
+                requests:
+                  cpu: "{{ .Values.cpu }}m"
 ```
 
-## Parameter Manipulation
+Note the inclusion of an `m` to specify millicpus.
 
-All parameters are suggested as integer values, sometimes it is necessary to manipulate a value to consume it in a patch. Patches are evaluated as [Go templates](https://golang.org/pkg/text/template/) with the added [Sprig](http://masterminds.github.io/sprig/) template functions. Additional template functions are also available:
+### Parameter Manipulation
+
+Sometimes it is necessary to manipulate a value to consume it in a patch. To support this, patches are evaluated as [Go templates](https://golang.org/pkg/text/template/), with [Sprig](http://masterminds.github.io/sprig/) template functions included to support a variety of operations. Additional template functions are also available:
 
 - **percent**
   Return the integer percentage.
 
-  `percent 9 50` will return `"4"`
+  `percent 9 50` will return `"4"`.
+
+## Parameter Constraints
+
+**Constraints** restrict the assignments that parameters may take relative to one another. Constraints are optional and used in addition to the bounds on individual parameters like `min` or `max`.
+
+Red Sky Ops supports two types of constraints: **Order** and **Sum**.
+
+### Order Constraint
+
+The order constraint requires that one integer parameter be strictly larger than another. For example, you can use an order constraint to ensure that a container's maximum replicas value is always greater than it's minimum replicas:
+
+```yaml
+  constraints:
+    - order:
+        lowerParameter: min_replicas
+        upperParameter: max_replicas
+```
+
+### Sum Constraint
+
+The sum constraint requires that the sum of two or more parameters not exceed an upper or lower bound. For example, in an experiment tuning CPU on multiple deployments, a sum constraint could enforce an overall cap of `4000` millicpus between both deployments:
+
+```yaml
+  constraints:
+    - sum:
+        bound: 4000
+        isUpperBound: true
+        parameters:
+          - name: first_cpu
+          - name: second_cpu
+```
+
+Each parameter in the sum constraint may have an associated **weight**. When a weight is specified for a parameter, the parameter's value is multiplied by that weight before being summed.
```

Comment:
If anyone has a good example usage of weighted sum constraints, I'll add it here.